039-AKSForEnterprise/Coach/01-containers.md

```diff
@@ -0,0 +1,117 @@
+# Challenge 1. Containers - Coach's Guide

```

Comment:
Again, make sure that the challenge name here, ie: "Containers", matches what you have in the coach README table of contents.

Also, it should be "Challenge #1: " ... change the period to a semicolon.

This applies to all the other Coach guide files too.

Reply:
But "Containers" already match, right? Replaced dot with colon!